libdnf5/rpm/solv/goal_private.cpp

```diff
@@ -376,6 +376,10 @@ libdnf5::GoalProblem GoalPrivate::resolve() {
     libsolv_solver.set_flag(SOLVER_FLAG_ALLOW_VENDORCHANGE, vendor_change);
     libsolv_solver.set_flag(SOLVER_FLAG_DUP_ALLOW_VENDORCHANGE, vendor_change);
 
+    // Ensure solver tries to use the latests version of dependecies even if it results in a bigger transaction
```

Comment:
Minor typo -> `Ensure the solver tries to use the latest version of dependencies, even if it results in a bigger transaction`